package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@quintype/build",
-  "version": "3.10.0",
+  "version": "3.10.1-fix-babelconfig.2",
```

Comment:
Revert this change and we are good to go.

Reply:
done 👍